### DISCUSSION TITLE
"Cancel" Unconfirmed outgoing txn. "Double Spend" or RBF to a new address.

### DISCUSSION THREAD

**Discussion by bitcoincoredotbin** [lightningnetwork/lnd#8316] [Troubleshooting]
I need to double spend on LND and I don't know how to do it.

Long story short, I screwed up twice in a row trying to pay an invoice. I scanned the on-chain bitcoin address instead of the LN invoice and it just broadcasted the txn with a low fee. I did this twice using Zeus wallet controlling my LND node. I know, I know, I should have been careful, but it's only like $30 total, so not a crazy amount. 

The fee was auto set at 2 sats per byte and broadcasted. I need to re-spend these UTXOs with a higher fee to an address I control. The existing addresses are to an expired invoice to a company. I can't wait it out and ask them for a refund because that could be months from now. Can anyone point me to a guide for a how to on spending an unconfirmed outgoing UTXO for LND? Seems like the lncli wallet bump fee command will only bump to the existing addresses. How can i construct a new txn with the UTXOs before this unconfirmed txn.

**Comment by bitromortac** [comment]
You could try https://github.com/lightninglabs/chantools/blob/master/doc/chantools_doublespendinputs.md
